dwi_ml/data/processing/streamlines/data_augmentation.py

```diff
@@ -3,73 +3,60 @@
 import nibabel as nib
 import numpy as np
 
+from dipy.io.stateful_tractogram import StatefulTractogram
 from scipy.stats import truncnorm
-from dwi_ml.data.processing.space.convert_world_to_vox import (
-    convert_world_to_vox)
+
 from dwi_ml.data.processing.streamlines.utils import split_array_at_lengths
 
 
-def add_noise_to_streamlines(
-        streamlines: Union[nib.streamlines.ArraySequence, np.ndarray],
-        noise_sigma: float, noise_rng: np.random.RandomState,
-        convert_mm_to_vox: bool = False, affine: np.ndarray = None)\
-        -> Union[nib.streamlines.ArraySequence, np.ndarray]:
+# Checked!
+def add_noise_to_streamlines(sft: StatefulTractogram,
```

Comment:
Would it make sense to have the option to do this in VOX space? This function is used mainly to do data augmentation on-the-fly to a batch of streamlines, and I wonder what the cost will be to send everything to RASMM and back...

Reply:
We should discuss this tomorrow.